exec/api/src/main/java/org/jboss/windup/engine/predicates/RuleProviderWithDependenciesPredicate.java

```diff
@@ -58,7 +58,7 @@ public boolean accept(WindupRuleProvider type)
                 List<String> executeAfterIDs = ruleProvider.getExecuteAfterIDs();
                 if ((executeAfter.contains(type.getClass())) || executeAfterIDs.contains(type.getID()))
                 {
-                    LOG.info("Accepting provider: " + type.getID());
+                    LOG.fine("Accepting provider: " + type.getID());
```

Comment:
Maybe we can leave this FINE and who needs that will turn on in logging.properties?